main/acle.rst

```diff
@@ -2227,22 +1662,17 @@ passed, so no PCS attributes are supported.
 
 The pcs attribute applies to functions and function types.
 Implementations are allowed to treat the procedure call specification as
-part of the type, i.e. as a language linkage in the sense of [C++
-#1].
+part of the type, i.e. as a language linkage in the sense of [C++ \#1].
```

Comment:
This [C++ \#1] does seem to make sense. The "good" news is that it does not render correctly or does not link to anythign even in previous versions of the ACLE (see for example Q32020 @ https://developer.arm.com/documentation/101028/0012/

Please create an issue just mentioning that the issue in this paragraph needs to be investigated and fixed.